recipe/helhum/typo3.php

```diff
@@ -178,17 +178,19 @@
 set('allow_anonymous_stats', false);
 
 /*
- * TYPO3 Specific config
+ * TYPO3-specific config
  */
 set('shared_dirs', [
     '{{typo3/root_dir}}/fileadmin',
     '{{typo3/root_dir}}/uploads',
     '{{typo3/root_dir}}/typo3temp/assets',
     '{{typo3/root_dir}}/typo3temp/var/locks',
+    'var/log',
 ]);
 
 set('shared_files',
     [
+        '.env',
```

Comment:
Whether .env needs to be a shared file or not, depends whether dotenv is used in production. Since it is hard to remove something from this array, I would tend to not add .env here

Reply:
Done